tests/data/invoice_assert_cases.py

```diff
@@ -0,0 +1,83 @@
+from advancedbilling.models.customer import Customer
+from advancedbilling.models.product import Product
+from advancedbilling.models.product_family import ProductFamily
+from advancedbilling.models.subscription import Subscription
+
+
+class InvoiceAssertCases:
```

Comment:
All these classes live in the `data` module and in general, provide only the pure dictionaries so, I'd think about different names instead of `assert` because there are no asserts there.